sonar-plugin-api/src/main/java/org/sonar/api/issue/action/Actions.java

```diff
@@ -25,7 +25,9 @@
 
 /**
  * @since 3.6
+ * @deprecated in 5.5. Webapp cannot be customized anymore to define actions on issues
  */
+@Deprecated
 @ServerSide
 public class Actions {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)